src/Column/ColumnNumber.php

```diff
@@ -34,12 +34,7 @@ public function getColumnValue(Row $row)
 			return $value;
 		}
 
-		return number_format(
-			(float) $value,
-			(int) $this->numberFormat[0],
-			(string) $this->numberFormat[1],
-			(string) $this->numberFormat[2]
-		);
+		return number_format((float) $value, (int) $this->numberFormat[0], (string) $this->numberFormat[1], (string) $this->numberFormat[2]);
```

Comment:
Please revert this line - 135 characters on 1 line seams too long for me. Thanks 👍

Reply:
Solved